kong/cli/cmds/config.lua

```diff
@@ -33,6 +32,12 @@ local DEFAULT_ENV_VALUES = {
       ["proxy_ssl_port"] = 8543,
       ["admin_api_port"] = 8101,
       ["dnsmasq_port"] = 8153,
+      -- TODO: This values are not properly dumped in enclosing quotes
+      --       which generates a wrong YAML value
+      --["cluster"] = {
+      --  ["bind"] = "0.0.0.0:7947",
+      --  ["rpc-addr"] = "127.0.0.1:7374"
+      --},
```

Comment:
This is currently a problem because of the YAML library we use. Related to https://github.com/lubyk/yaml/issues/2.